test/testapps/BasicTestApp/RouterTest/Links.cshtml

```diff
@@ -20,3 +20,6 @@
 <a id="anchor-with-no-href">
     Anchor tag with no href attribute
 </a>
+
+<a href="/" target="_blank">Target (_blank)</a>
+<a href="" target="_self">Target (_self)</a>
```

Comment:
Why do these two `<a>` tags have different `href` values? What distinction is being tested?

Reply:
1) href="/" opens the link "/" new tabs
2) href="" reloads the page current tab

Reply:
Sorry if I was unclear. I'm asking *why* you're doing that. Why does the `"/"` go with `_blank` and the `""` go with `_self`?